### ISSUE TITLE
Component: Form error

### ISSUE THREAD

**Issue by usethetics** [nl-design-system/backlog#122]
- [ ] HTML/CSS component 
- [ ] Web Component
- [ ] Figma component
- [ ] Design tokens
- [ ] Documentation

**Comment by julezrulez** [comment]
HTML5 validation does not conform to WCAG at the moment. In some browsers it is getting better, but better is not good enough.
Some of the problems I see in the world wild web:
- Errors are in English in an English browser, while the page is in Dutch and has a lang="nl". (indirectly WCAG)
- Error message pops up at the field with the error and stays there only for several seconds. (WCAG)
- Error message is gone as soon the field gets focus, so it could be possibly hard to correct your error. (WCAG)
- Error message popup in a way that they are outside the viewport. 
- Texts or error message are not complete enough or it is an suggestion or instruction instead of an error message. (WCAG)
- Lack of suggestions or instructions (WCAG)
The list above might not be complete. Also there are sources on the internet that keep saying HTML5 validation is accessible en onforms to WCAG. They just have to proof me I am wrong.

**Comment by usethetics** [comment]
Thanks @julezrulez, this is valuable information to document in the guidelines!

**Comment by wartburggraaf** [comment]
Amsterdam: https://github.com/Amsterdam/design-system/blob/main/packages/css/src/components/error-message/error-message.scss
Utrecht: https://github.com/nl-design-system/utrecht/tree/main/components/form-field-error-message

Op Storybook als onderdeel van Form field: https://nl-design-system.github.io/themes/?path=/story/utrecht-form-field--voorbeeld-theme

**Comment by jeffreylauwers** [comment]
## Varianten

We hebben meerdere implementaties van Form Field Error Message bekeken en met elkaar vergeleken. Op basis daarvan hebben we besloten we dat er géén specifieke varianten nodig zijn voor dit Candidate component.

**Comment by jeffreylauwers** [comment]
## Anatomie

<img width="960" height="540" alt="Een Form Field Error Message met aanwijzers. De kleur van de Form Field Error Message is rood. Een aanwijzer met het cijfer 1 wijst naar de Form Field Error Message in zijn geheel. Een aanwijzer met het cijfer 2 wijst naar een icoon in de vorm van een cirkel met daarbinnen een uitroepteken, dit icoon staat voorafgaand aan de tekst. Een aanwijzer met het cijfer 3 wijst naar de tekst 'Dit is een voorbeeld van een Form Field Error Message.  En in een Form Field Error Message kunnen ook
twee alinea’s zitten.'." src="https://github.com/user-attachments/assets/b374fd3c-a0c3-4975-b9c2-e013991901a0" />

De component bestaat uit:
1. Form Field Error Message
2. Icon (optioneel): voor het plaatsen van een icoon voorafgaand aan de tekst.
3. Content: voor de inhoud in de Form Field Error Message.

[Deze schets is gemaakt in Figma](https://www.figma.com/design/0Y9Tbb373u6QGxGT6mqUnm/NL-Design-System---Doc-website---Afbeeldingen?node-id=1777-797)
